spec/dummy/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationController < ActionController::Base
```

Comment:
I think there are templates for rails, which is one file as an entire rails application. I know @derekprior have mentioned it at some point, but I can't find it now.